Firmware/mmu2.cpp

```diff
@@ -608,19 +608,23 @@ void MMU2::ReportError(ErrorCode ec) {
     if( ec != lastErrorCode ){ // deduplicate: only report changes in error codes into the log
         lastErrorCode = ec;
 
-        // Log error format: MMU2:E=32766 TextDescription
-        char msg[64];
-        snprintf(msg, sizeof(msg), "MMU2:E=%hu", (uint16_t)ec);
+        // Log error format: MMU2:E=32766 ErrorTitle TextDescription
+
+        // The longest error description in errors_list.h is 144 bytes.
```

Comment:
this is subject for removal - it was kept here just to prevent the compiler from stripping the texts completely.